pkg/ipmi/ipmi.go

```diff
@@ -77,6 +89,12 @@ type systemInterfaceAddr struct {
 	lun      byte //nolint:unused
 }
 
+type setSystemInfoReq struct {
+	paramSelector byte
+	setSelector   byte
+	strData       [_SYSTEM_INFO_BLK_SZ]byte
```

Comment:
NIT: Do we need a constant for this?

Reply:
Do you mean _SYSTEM_INFO_BLK_SZ?  Maybe change it to var?

Reply:
I meant to say why not directly do `strData [16]byte` but it is fine, lets leave it leave this.